src/main/java/org/trigon/hardware/rev/spark/SparkInputs.java

```diff
@@ -28,19 +33,30 @@ public void toLog(LogTable table) {
         latestTable = table;
     }
 
+    /**
+     * Registers a signal.
+     *
+     * @param statusSignal the signal to register
+     */
     public void registerSignal(SparkStatusSignal statusSignal) {
         if (statusSignal == null || RobotHardwareStats.isReplay())
             return;
 
         addSignalToSignalsArray(statusSignal);
     }
 
+    /**
+     * Registers a threaded signal.
+     * Threaded signals use threading to process certain signals separately. Should be used for signals that need to be updated at a high frequency such as odometry.
```

Comment:
Threaded signals use threading to process certain signals separately at a faster rate.
Keep the jdoc consistent across all threaded stuff